front/src/modules/infra/styles/_infraSelector.scss

```
@@ -127,6 +127,65 @@
       line-height: 20px;
       font-weight: 500;
     }
+
+    .infra-selector-grants-manager {
+      margin: 0 16px 8px;
+      padding: 5px 0 7px;
+      border-bottom: 1px solid var(--grey20);
+      button:focus {
+        outline: none;
+      }
+    }
+    .infra-selector-grants-header {
+      display: flex;
+      justify-content: space-between;
+      .user-infra-grant {
+        font-size: 0.875rem;
+        color: var(--info60);
+      }
+      .blue-label {
+        color: var(--primary60);
+      }
+      .grey-label {
+        color: var(--grey30);
+      }
+    }
+    .infra-selector-subject {
+      &-list {
+        display: flex;
+        flex-direction: column;
+        border-radius: 6px;
+        margin: 15px 8px 8px;
+        background-color: var(--ambientC10);
+      }
+      &-card {
+        display: flex;
+        justify-content: space-between;
+        height: 47px;
+        margin: 0 8px;
```

You can simplify with 
```suggestion
        margin-inline: 8px;
```